vm-setup/roles/packages_installation/tasks/main.yml

```diff
@@ -46,24 +46,17 @@
   become: yes
   when: ansible_facts['distribution'] == "Ubuntu"
 
-- name: Install packages on CentOS/RHEL8
+- name: Install packages on CentOS/RHEL
   block:
     - name: Install packages on CentOS/RHEL
       package:
           name: "{{ packages.centos.common.packages }}"
           state: present
-    - name: Install packages on CentOS8/RHEL8
-      package:
-          name: "{{ packages.centos.el8.packages }}"
```

Comment:
I suspect we can now consolidate the el8 and el9 lists (at least until el10 is a thing, which is relatively soon lol)

Reply:
that's what blocked me from doing it

Reply:
yeah, but we still don't need el8 packages any more

Reply:
yep, that's what I forgot, thanks